blog/2020-09-25-gh-release.md

```diff
@@ -0,0 +1,165 @@
+ A nice benefit of using GitFlow is that we could actually have a dedicated release-CI workflow that performs more exhaustive checks and is triggered by any push to a `release/*` branch.
+ We are not doing that yet but it is something to keep in mind.
+
+By using a dedicated bot user, we can work around this limitation and our CI workflow is properly triggered after the bot pushes the release branch.
+
+To use a dedicated bot user:
+
+1. create a personal access token for that user
+2. create a secret that is accessible to the repository that contains your workflow
+3. reference the secret in the required workflow steps: https://github.com/comit-network/comit-rs/blob/acabab36058b4e3d3ae54bde10fd7ce5db66f1a5/.github/workflows/create-cnd-gh-release.yml#L30
+
+GitHub actually supports [organization-wide](https://github.blog/2020-05-22-keep-your-secrets-synced-across-multiple-repositories-with-organization-secrets/) secrets.
+If you don't want to create a new token for every one of your repositories, you can store the token as an organization secret and it will be accessible to all repositories.
+
+## Using `workflow_dispatch` triggers
+
+In a previous version of our release process, we used issues as the initial trigger.
+I wrote about this on [my personal blog](https://blog.eizinger.io/12274/using-github-actions-and-gitflow-to-automate-your-release-process) in detail in case you are interested.
+
+Fortunately, GitHub has since moved on and [added](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) the `workflow_dispatch` event.
+This allows us to create a workflow with user input that can be triggered at the click of a button.
+
+As you might have already guessed, this is what we [did](https://github.com/comit-network/comit-rs/blob/acabab36058b4e3d3ae54bde10fd7ce5db66f1a5/.github/workflows/draft-new-cnd-release.yml#L3-L8).
+Given a version, the user with write access can trigger the release process which kicks-off the release process.
+
+Personally, I find the ability to trigger workflows this way pretty cool as it opens up a lot of opportunities to automate aspects of your development cycle.
+
+## Modular workflows
+
+Our release process is split into three different workflows:
+
+1. The initial workflow, triggered by the `workflow_dispatch` event
+2. Creating a GitHub release, triggered by the release-branch being merged into `master`
+3. Building the binaries and attaching them, triggered by the `release` event
+
+This separation has several nice benefits:
+
+### Maintainability
+
+It makes the individual files smaller, making them easier to understand and maintain.
+
+### Reusability
+
+Modular workflows are easier to reuse.
+For example, we recently [added](https://github.com/comit-network/ambrosia/pull/82) automated releases to Ambrosia - a frontend for cnd that we are currently developing.
```

Comment:
```suggestion
For example, we recently [added](https://github.com/comit-network/ambrosia/pull/82) automated releases to [Ambrosia](https://github.com/comit-network/ambrosia) - a frontend for cnd that we are currently developing.
```

Shall we maybe introduce the repos at some point? 

You keep talking about Ambrosia and comit-rs, I think it should be somehow mentioned what they are. 
Not that it is important in the context of this post, but in general I would tell the user.